admin/tools/docker/tagged/Dockerfile.tpl

```diff
@@ -3,16 +3,17 @@ MAINTAINER Fabrice Jammes <fabrice.jammes@in2p3.fr>
 
 USER root
 
-RUN apt-get update && apt-get -y install byobu git vim gdb lsof
+RUN apt-get update
+RUN apt-get -y install byobu git vim gdb lsof
 
 USER qserv 
 
 WORKDIR /home/qserv
 
-RUN mkdir src && cd src && git clone -b {{GIT_TAG_OPT}} --single-branch https://github.com/LSST/qserv
-WORKDIR /home/qserv/src/qserv
-RUN bash -c ". /qserv/stack/loadLSST.bash && setup -r . -t qserv-dev && eupspkg -er build"
+RUN mkdir src
+WORKDIR /home/qserv/src
 # Force execution of lines below by changing timestamp
-RUN git pull # Build performed on {{TIMESTAMP}}
+RUN git clone -b {{GIT_TAG_OPT}} --single-branch https://github.com/LSST/qserv # Build performed on {{TIMESTAMP}}
```

Comment:
Do we really need/want a full git clone (all history, branches, metadata etc.) in our docker images?  Maybe we should try something like this instead, to just get the tip of the source tree:

    git archive --format=tar --remote=<repository URL> {{GIT_TAG_OPT}} | tar xf -